packages/swarm-styles/lib/components/numericalInput.css

```diff
@@ -42,10 +44,30 @@
 }
 
 [data-swarm-number-input-field] {
+    width: 100%;
     height: 16px;
     margin: 0;
 }
 
+[data-swarm-number-input-increment] {
+    margin-left: var(--space-half);
+}
+
+/* hover */
+[data-swarm-number-input]:hover {
+   border-color: var(--color-gray-6);
+}
+
+[data-swarm-number-input] button:hover {
+    cursor: pointer;
+}
+
+/* placeholder */
+[data-swarm-number-input-field]::placeholder {
```

Comment:
I think this is a typo - there are two`:` instead of one.

Reply:
It's a pseudo element so it needs two. more here -> https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder